### ISSUE TITLE
undefined fireDate

### ISSUE THREAD

**Issue by LouisWayne** [node-schedule/node-schedule#410]
The code is not working as intended, here is the sample code:

```
const j = NodeSchedule.scheduleJob('0 * * * * *', function(fireDate) {
    console.log('This job was supposed to run at ' + fireDate + ', but actually ran at ' + new Date());
});
```

The job in the code will be triggered every minute - but `fireDate` is undefined.  I am using the version: `1.2.5`.

**Comment by MichelJansson** [comment]
I have the same issue and saw that the code for this was not there.
This feature/code have not been published to npm yet as far as i can tell.

**Comment by santigimeno** [comment]
Just published `1.3.0`. It should work there.
